blocks/feed/feed.js

```diff
@@ -139,7 +152,15 @@ export default async function decorate(block) {
     .take(blockCfg.count ? parseInt(blockCfg.count, 10) : 4)
     .toList();
   block.innerHTML = '';
-  const blockContents = resultParsers[blockType](results, blockCfg);
+  if (block.classList.contains('featured')) {
```

Comment:
`featured` should be present in `blockCfg` already. Authors should specify it in block config instead of a variant.

Reply:
This featured is not of the one in blockCfg. This featured is of the block name - "feed (featured)", since there are certain unique changes to be done for latest featured card & not to every featured card.

Reply:
ok, I feel we are overloading the highlight block with too many things here. Looking at the page, I am wondering if this should a plain columns block block. Is there a reason we need it to be a feed block?